<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorEditView</class>
 <widget class="QDialog" name="DoctorEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医疗信息管理系统</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_32">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>编辑医生信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>32</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="DoctorName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>性别：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Sex">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>男</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>女</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_25">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>民族：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Ration">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>汉</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>壮</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>维吾尔</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>回</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>苗</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>满</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>彝</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>土家族</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>藏族</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>蒙古族</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>入职时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="EntryTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>出生日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="Born">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>年龄：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Age">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>身高：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Height">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>学历：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="EducationBackground">
           <item>
            <property name="text">
             <string>本科</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>硕士</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>博士</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>婚姻状况：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="MaritalStatus">
           <item>
            <property name="text">
             <string>未婚</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>已婚</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>离婚</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>丧偶</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>体重：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Weight"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>工号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="JobId">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>所在部门：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Department">
           <item>
            <property name="text">
             <string>全科诊室</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>治疗室</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>处置室</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>预防保健室</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>健康信息管理室</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>职务：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="JobIdentity">
           <item>
            <property name="text">
             <string>医士</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>住院医师</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>主治医师</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>副主任医师</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>主任医师</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>证件类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CertificateType">
             <item>
              <property name="text">
               <string>中华人民共和国居民身份证</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>台湾居民来往大陆通行证</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>港澳居民来往内地通行证</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>护照</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>香港身份证</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>澳门身份证</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>中华人民共和国外国人永久居留身份证</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>港澳居民居住证</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>台湾居民居住证</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>证件号码：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="CertificateNumber"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>籍贯：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NativePlace"/>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>家庭住址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Place"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>联系电话：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Phone"/>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>电子邮箱：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Email"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>执业证书号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PractiseCertificate"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="btSaveDoctorMessage">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btCancelDoctorMessage">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
